### DISCUSSION TITLE
Styles with Python code

### DISCUSSION THREAD

**Discussion by mauf3r** [Avaiga/taipy#1547] [Q&A]
I am trying to create my first app, using the python approach. And I found, the docs on styles are mostly focussed on Markdown.

```python
            with tgb.layout('5*1', gap='2.0rem'):
                card_style = 'card pb0'
                heading_style = 'text-right h5 color-primary'
                number_style = 'text-right h1 pb0'
                with tgb.part(card_style):
                    tgb.text("**Headline**", class_name=heading_style, mode='md')
                    tgb.text("{x['a']['b']['']}", format='%.0f', class_name=number_style, mode='md')
```

The formatting of the text() elements is working after some try and error.
My aesthetic issue: the space between the text and end of the card is huge.
I am no HTML expert, but I think that's called padding. I found how to adjust padding in the Stylekit section of the manual, but my above settings don't have any impact.

**Comment by FlorianJacta** [comment]
This is because of the `mode="md"`. This prevents the *class_name* to apply. I will create an issue on Taipy to solve this!

**Reply by FlorianJacta** [reply]
https://github.com/Avaiga/taipy/issues/1553

**Reply by mauf3r** [reply]
but the class_name in tgb.text() is working.
The class_name in tab.part() is not, and there is no mode='md' 🤔

**Reply by FlorianJacta** [reply]
Try to delete the mode='md'. It deletes the space your are talking about. The card is not the one responsible for the padding. You will see this:

![image](https://github.com/user-attachments/assets/3230de7e-34d9-4121-8cea-d9f5e4b3530a)
